core/src/apps/cardano/layout.py

```diff
@@ -122,8 +122,8 @@ async def show_warning_tx_different_staking_account(
     page1.normal("the current account.")
 
     page2 = Text("Confirm transaction", ui.ICON_SEND, ui.GREEN)
-    page2.normal("Staking account:")
-    page2.bold("#%d" % to_account_number(staking_account_path))
+    page2.normal("Staking account #%d:" % to_account_number(staking_account_path))
```

Comment:
I would refactor `to_account_number` to `format_account_number` and return the account number alongside the hashtag, to enforce consistency when displaying it to the user. Or do you see a reason not to?

Reply:
I've left the `to_account_number` function there, just to separate it and I added a `to_formatted_account_number` function, which is the one used in `layout.py`.